Fix mute script: save volume and then set it to 0 instead of saving it directly 
to 0. Bug #122087.
Index: aumix-2.8/src/mute
===================================================================
--- aumix-2.8.orig/src/mute
+++ aumix-2.8/src/mute
@@ -8,7 +8,8 @@
 volumes=$(aumix -vq |tr -d ,)
 if [ $(echo $volumes | awk '{print $2}') -ne 0 -o \
 	$(echo $volumes | awk '{print $3}') -ne 0 ]; then
-	aumix -S -v 0
+	aumix -S
+	aumix -v 0
 else
 	aumix -L > /dev/null
 fi
